### PR TITLE
WIP: Implementing the primary DGEList class

### PR DESCRIPTION
**Pull request by clintval**
EDIT: Do not review / merge.

### CODE REVIEW

File: edgePy/DGEList/DGEList.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import numpy as np
+
```
Comment (clintval) [inline edgePy/DGEList/DGEList.py]: Add:

```python
__all__ = [
    'DGEList'
]

File: edgePy/DGEList/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+from .DGEList import *
```
Comment (clintval) [inline edgePy/DGEList/__init__.py]: Do relative import (see next comment too):

```python
from ._DGEList import *
```

File: edgePy/__init__.py [hunk 3]
```diff
@@ -1 +1 @@
-from edgePy import io
+from edgePy import io, DGEList
```
Comment (clintval) [inline edgePy/__init__.py]: Make two lines for now:

```python
from edgePy import io
from edgePy import DGEList 
```

File: edgePy/io/__init__.py [hunk 4]
```diff
@@ -1 +1 @@
-from ._io import *
+from ._io import *
```
Comment (clintval) [inline edgePy/io/__init__.py]: Keep newline please?

File: tests/test_DGEL.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import pkgutil
+
+from nose.tools import eq_
+
+from unittest import TestCase
+
+from _io import *
```
Comment (clintval) [inline tests/test_DGEL.py]: Make this:

```python
from edgePy.io import *
```

File: tests/test_DGEL.py [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+from DGEList import *
```
Comment (clintval) [inline tests/test_DGEL.py]: Do this:

```python
from edgePy.DGEList import *
```

Comment (eegk) [reply tests/test_DGEL.py]: interesting.

**Comment by TrackAddict** [comment]
I just set up the Ubuntu Subsystem on Windows today, and I was grabbing the files from the Windows side but manipulating (and committing) them in the subsystem. Maybe that had something to do with it?

I'll get more traditional dual-booting going in the near future and see if that helps. Actual modification happened in _DGEList.py but nowhere else that I'm aware of.

**Comment by clintval** [comment]
Ah the Windows platform makes sense.

You ended up changing all lines to CRLF line endings. There should be a setting in your Git client to turn on auto-CRLF line ending handling otherwise you will do this for every file you touch 😄 

Can read this:

https://help.github.com/articles/dealing-with-line-endings/

**Comment by TrackAddict** [comment]
Oh man, I didn't even think about that, great catch!

I'll definitely read through that, although I don't plan on pushing anything from Windows anymore. I've got a Mac at work and I can dual boot at home. Will get that changed and re-push as soon as I've got a minute.

**Pull request closed without merging**